local-setup/scripts/enable-dashboard.sh

```
@@ -0,0 +1,921 @@
+      local sc="${entry%%/*}" uid="${entry##*/}"
+      if [[ "$DRY_RUN" == true ]]; then log "[dry-run] deactivate $sc/$uid"; continue; fi
+      local cur body resp
+      cur="$(curl -s -X POST "$MDMS_URL/mdms-v2/v2/_search/$sc" -H 'Content-Type: application/json' \
+        -d "{\"RequestInfo\":$REQUEST_INFO,\"MdmsCriteria\":{\"tenantId\":\"$DASHBOARD_TENANT\",
+             \"schemaCode\":\"$sc\",\"uniqueIdentifiers\":[\"$uid\"]}}")"
+      body="$(mktemp)"
+      jq -n --argjson ri "$REQUEST_INFO" --argjson cur "$cur" \
+        '{RequestInfo:$ri, Mdms:($cur.mdms[0] + {isActive:false})}' > "$body" 2>/dev/null || {
+          warn "could not read $sc/$uid for repair — deactivate it by hand"; rm -f "$body"; continue; }
+      resp="$(mdms_post "/mdms-v2/v2/_update/$sc" "$body")"; rm -f "$body"
+      printf '%s' "$resp" | grep -qi 'mdms' && ok "deactivated corrupt $sc/$uid" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)